src/pages/scan/scan.ts

```diff
@@ -59,6 +65,7 @@ export async function handleEvent(
   const server = await getServer({baseUrl: data.server.baseUrl});
   if (server) {
     // No need to perform authentication
+    await updateServer(server.id, data.customCodeHandlers);
```

Comment:
I think you can nonetheless check using something like `_.isEqual()` if we have lodash available in the app. Unless just always updating the server is fine as well. cc @tomasr8

Reply:
I'd say that's overkill for this